pkg/core/blockchain_metrics.go

```diff
@@ -0,0 +1,66 @@
+package core
+
+import "github.com/prometheus/client_golang/prometheus"
+
+// Metrics for monitoring service.
+var (
+	//BlockHeight prometheus metric.
+	BlockHeight = prometheus.NewGauge(
```

Comment:
And now we have lots of (exported!) globals again. Why not put them into some kind of structure? What I want to see in the end result is something like
```
bcMetrics.blockHeight.Set()
```
or something similar.

Reply:
but in previous iteration I had a structure with `Update(..)` method. What was wrong with that?

Reply:
It was like
```go
smth := &fakeStruct{}
smth.Update()
```
which looks like some local thingie created, updated and then thrown away, that's not what we're doing here. We're changing something in the package's state, so it should look like a change in the package's state, it's just that it shouldn't be tens of state variables but rather one well-known state structure with a nice name.

Reply:
And it should be this way for the other packages too, of course.

Reply:
Another option is to expose a set of functions to update metrics, this is what is done by the neofs-node in many places, but that has a little more overhead to implement.

Reply:
And while we're at it, it would also be nice to have all prometheus variables and initialization done in one `*.go` file in each of the packages, `metrics.go` or `prometheus.go`, so that it would be uniformly done across the codebase.

Reply:
added this way. Added functions to update metrics. But want to mention that in neo-fs they have function to update metric only in case of complicated scenario. In simple operations like `Inc()` like in `events` with `irEventsCount` it's taken as is.